remote_llm.py

```diff
@@ -32,15 +35,20 @@ def generate_text(preprompt, prompt, model):
     if token is None:
         raise ValueError('LLM Verification Router token is not set')
     
-    log.info(f'Received text generation request for prompt "{prompt}" for model {model}')
+    log.error(f'Received text generation request for prompt "{prompt}" for model {model} at url {url}')
     # Load the Vanilla Neox API key from the config file.
 
     raw_response = post(url=route,
                         headers={'Authorization': f'Bearer {token}'},
                         json={'prompt': prompt, "preprompt" : preprompt, "model": model})
+
+    try:
```

Comment:
Provides a much better exception message than the json decode I was getting before with the bad llmv_router